### PR TITLE
text change in disco pane

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: src/olympia/discovery/data.py [hunk 1]
```diff
@@ -19,8 +19,8 @@ def __init__(self, *args, **kwargs):
                   '{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'Remove blaring ads and make the Web run smoother! Adblock '
-              u'Plus can also mitigate tracking mechanisms and malware.'),
+            _(u'Remove blaring ads and make the Web run smoother with ')
```
Comment (mstriemer) [inline src/olympia/discovery/data.py]: The `)` on the end of this line looks like it will cause a syntax error.

**Comment by muffinresearch** [comment]
This change might mean some locales fall-back to english if we don't get updates before the tag.

**Comment by devaneymoz** [comment]
The English fall-back will be okay in this case. Thanks!

File: src/olympia/discovery/data.py (hunk 1 above)
Comment (diox) [inline src/olympia/discovery/data.py]: It won't, but that means the second part of the string is not part of the gettext call, which is bad.

**Comment by TonnesM** [comment]
Does this have a follow-up in order to get new l10n content asap?

**Comment by EnTeQuAk** [comment]
Just opened a PR for extracting the neccessary strings.

File: src/olympia/discovery/data.py (hunk 1 above)
Comment (diox) [inline src/olympia/discovery/data.py]: Was fixed in https://github.com/mozilla/addons-server/commit/0c2b633c592a882a51698949f2262773d1462b5b

**Pull request merged**